<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>750</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Segoe UI</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>EquivSolver</string>
  </property>
  <property name="statusTip">
   <string/>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextUnderIcon</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="FieldView" name="circuitGraphicsView">
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="windowTitle">
    <string>Field tools</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSelection"/>
   <addaction name="actionDraw"/>
   <addaction name="actionDelete"/>
   <addaction name="separator"/>
   <addaction name="actionI"/>
   <addaction name="actionE"/>
   <addaction name="actionR"/>
   <addaction name="actionC"/>
   <addaction name="actionL"/>
   <addaction name="actionGnd"/>
   <addaction name="separator"/>
   <addaction name="actionSolve"/>
  </widget>
  <widget class="WindowFrameMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>950</width>
     <height>23</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <property name="nativeMenuBar">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuElements">
    <property name="title">
     <string>Elements</string>
    </property>
    <addaction name="actionI"/>
    <addaction name="actionE"/>
    <addaction name="actionR"/>
    <addaction name="actionC"/>
    <addaction name="actionL"/>
    <addaction name="actionGnd"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionLoad"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionSelection"/>
    <addaction name="actionDraw"/>
    <addaction name="actionDelete"/>
    <addaction name="separator"/>
    <addaction name="actionRotateElementRight"/>
    <addaction name="actionRotateElementLeft"/>
   </widget>
   <widget class="QMenu" name="menuSolver">
    <property name="title">
     <string>Solver</string>
    </property>
    <addaction name="actionSolve"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionCircuitSettings"/>
    <addaction name="separator"/>
    <addaction name="actionFieldSettings"/>
   </widget>
   <widget class="QMenu" name="menuTests">
    <property name="title">
     <string>Debug</string>
    </property>
    <addaction name="actionGraphicsItemDebug"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuElements"/>
   <addaction name="menuSettings"/>
   <addaction name="menuSolver"/>
   <addaction name="menuTests"/>
  </widget>
  <widget class="QDockWidget" name="dockWidgetProperties">
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>64</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Model properties</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetPropertiesContents">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>10</number>
     </property>
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="topMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <property name="bottomMargin">
      <number>10</number>
     </property>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionI">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="application.qrc">
     <normaloff>:/circuit_icons/resources/current.png</normaloff>:/circuit_icons/resources/current.png</iconset>
   </property>
   <property name="text">
    <string>I</string>
   </property>
   <property name="toolTip">
    <string>I (Currency Source)</string>
   </property>
   <property name="shortcut">
    <string>Shift+I</string>
   </property>
  </action>
  <action name="actionE">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="application.qrc">
     <normaloff>:/circuit_icons/resources/voltage.png</normaloff>:/circuit_icons/resources/voltage.png</iconset>
   </property>
   <property name="text">
    <string>E</string>
   </property>
   <property name="toolTip">
    <string>E (Voltage Source)</string>
   </property>
   <property name="shortcut">
    <string>Shift+E</string>
   </property>
  </action>
  <action name="actionR">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="application.qrc">
     <normaloff>:/circuit_icons/resources/resistor.png</normaloff>:/circuit_icons/resources/resistor.png</iconset>
   </property>
   <property name="text">
    <string>R</string>
   </property>
   <property name="toolTip">
    <string>R (Resistor)</string>
   </property>
   <property name="shortcut">
    <string>Shift+R</string>
   </property>
  </action>
  <action name="actionC">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="application.qrc">
     <normaloff>:/circuit_icons/resources/capacitor.png</normaloff>:/circuit_icons/resources/capacitor.png</iconset>
   </property>
   <property name="text">
    <string>C</string>
   </property>
   <property name="toolTip">
    <string>C (Capacitor)</string>
   </property>
   <property name="shortcut">
    <string>Shift+C</string>
   </property>
  </action>
  <action name="actionL">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="application.qrc">
     <normaloff>:/circuit_icons/resources/inductor.png</normaloff>:/circuit_icons/resources/inductor.png</iconset>
   </property>
   <property name="text">
    <string>L</string>
   </property>
   <property name="toolTip">
    <string>L (Inductor)</string>
   </property>
   <property name="shortcut">
    <string>Shift+L</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save to file</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>Save As ...</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Load from file</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="menuRole">
    <enum>QAction::QuitRole</enum>
   </property>
  </action>
  <action name="actionSelection">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="application.qrc">
     <normaloff>:/tools_icons/resources/tools/select.png</normaloff>:/tools_icons/resources/tools/select.png</iconset>
   </property>
   <property name="text">
    <string>Selection</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
  <action name="actionDraw">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="application.qrc">
     <normaloff>:/tools_icons/resources/tools/pencil.png</normaloff>:/tools_icons/resources/tools/pencil.png</iconset>
   </property>
   <property name="text">
    <string>Draw wire</string>
   </property>
   <property name="toolTip">
    <string>Draw wire</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="application.qrc">
     <normaloff>:/tools_icons/resources/tools/erase.png</normaloff>:/tools_icons/resources/tools/erase.png</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionSolve">
   <property name="icon">
    <iconset resource="application.qrc">
     <normaloff>:/tools_icons/resources/tools/graph.png</normaloff>:/tools_icons/resources/tools/graph.png</iconset>
   </property>
   <property name="text">
    <string>Solve</string>
   </property>
  </action>
  <action name="actionGnd">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="application.qrc">
     <normaloff>:/circuit_icons/resources/ground.png</normaloff>:/circuit_icons/resources/ground.png</iconset>
   </property>
   <property name="text">
    <string>Gnd</string>
   </property>
   <property name="toolTip">
    <string>Ground</string>
   </property>
   <property name="shortcut">
    <string>Shift+G</string>
   </property>
  </action>
  <action name="actionFieldSettings">
   <property name="text">
    <string>Field Settings</string>
   </property>
  </action>
  <action name="actionCircuitSettings">
   <property name="text">
    <string>Circuit settings</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New circuit</string>
   </property>
  </action>
  <action name="actionGraphicsItemDebug">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>GraphicsItem debug</string>
   </property>
   <property name="toolTip">
    <string>GraphicsItem debug</string>
   </property>
  </action>
  <action name="actionRotateElementRight">
   <property name="text">
    <string>Rotate element right</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionRotateElementLeft">
   <property name="text">
    <string>Rotate element left</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+R</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>FieldView</class>
   <extends>QGraphicsView</extends>
   <header>field/fieldview.h</header>
  </customwidget>
  <customwidget>
   <class>WindowFrameMenuBar</class>
   <extends>QMenuBar</extends>
   <header>custom_widgets/windowframemenubar.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="application.qrc"/>
 </resources>
 <connections/>
</ui>
